src/js/client/handlebars.js

```diff
@@ -0,0 +1,179 @@
+// A client-side module that provides various template handling capabilities, and which wires in any
+// child components with the grade `gpii.templates.hb.helper` as handlebars helpers.
+//
+// Requires Handlebars.js and Pagedown (for markdown rendering)
+
+/* global fluid, jQuery, Handlebars */
+(function ($) {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.templates.hb.client");
+
+    gpii.templates.hb.client.addHelper = function (that, component) {
+        var key = component.getHelperName();
```

Comment:
I think all of "getHelperName()" can  be removed throughout. It's perfectly fine to reference the relevant component option directly via component.options.name (or component.options.helperName if you prefer for clarity)